astropy/io/fits/verify.py

```diff
@@ -1,4 +1,4 @@
-# Licensed under a 3-clause BSD style license - see PYFITS.rst
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
```

Comment:
This header is present at the top of almost all files in `astropy.io.fits`:
```
❯ grep -ir PYFITS.rst astropy/io/fits | wc -l
32
```
The license is the same (3 clause BSD), the copyright holder is not, not sure if it makes sense to keep the old license ?